apps/fluent-tester/src/FluentTester/TestComponents/RadioGroup/RadioGroupTest.tsx

```diff
@@ -35,8 +35,8 @@ const basicRadioGroup: React.FunctionComponent<{}> = () => {
       <Separator />
       <RadioGroup label="Controlled RadioGroup" selectedKey={selectedKey} onChange={onChange2}>
         <RadioButton content="Option A" buttonKey="A" ariaLabel="Test Aria Label" />
-        <RadioButton content="Option B" buttonKey="B" />
-        <RadioButton content="Option C" buttonKey="C" />
+        <RadioButton content="Option B" buttonKey="B" ariaPosInSet={57} />
+        <RadioButton content="Option C" buttonKey="C" ariaSetSize={18} />
```

Comment:
While these values are inaccurate, this test aims to show someone can override the auto-generated values. A screen reader reading this would say "57 out of 4" or "3 out of 18".

Reply:
Could you add a comment about that in the code so that we understand what it's there for in the future?

Reply:
Added one